intelmq/bots/parsers/alienvault/parser_otx.py

```diff
@@ -26,16 +27,34 @@ def process(self):
         raw_report = utils.base64_decode(report.get("raw"))
 
         for pulse in json.loads(raw_report):
-            additional = {"author": pulse['author_name'], "pulse": pulse['name']}
+            additional_pulse = {"author": pulse['author_name'],
+                                "pulse": pulse['name']}
+
             for indicator in pulse["indicators"]:
+                additional_indicator = {}
                 event = self.new_event(report)
                 # hashes
                 if indicator["type"] in HASHES.keys():
                     event.add(HASHES[indicator["type"]], indicator["indicator"])
+                    additional_indicator['malware.hash.type'] = indicator["type"]
```

Comment:
Let me revisit my use case, this was before I read the DHO discussion around malware hashes. In couple of hours, I will either remove this part of code or add a use case for having it.